src/openmmio.py

```diff
@@ -1168,10 +1247,7 @@ def multipole_moments(self, shot=0, optimize=True, polarizability=False):
             logger.error("Polarizability calculation is available in TINKER only.\n")
             raise NotImplementedError
 
-        if self.platname in ['CUDA', 'OpenCL'] and self.precision in ['single', 'mixed']:
-            crit = 1e-4
-        else:
-            crit = 1e-6
+        crit = 1e-4 if (self.platname in ['CUDA', 'OpenCL'] and self.precision in ['single', 'mixed']) else 1e-6
```

Comment:
Stylistically, this was a bit easier to understand before the change.

Reply:
I've reverted this in the latest commit.